src/pykeen/evaluation/evaluator.py

```diff
@@ -67,11 +67,13 @@ def __init__(
         requires_positive_mask: bool = False,
         batch_size: int = None,
         slice_size: int = None,
+        automatic_memory_optimization: bool = True,
```

Comment:
update the init's for the subclasses of evaluator too to make sure that this can be passed through

Reply:
Done in f6331dab